pandokia/top_level.html

```diff
@@ -1,48 +1,38 @@
 PAGEHEADER
 
-<p>
+<!--<p>
 Documentation is available on the group wiki: 
 <a href="https://svn.stsci.edu/trac/ssb/development/wiki/Pandokia">
 https://svn.stsci.edu/trac/ssb/development/wiki/Pandokia
 </a>
+</p>-->
 
-<br/><br/>
-
-Logged in as: <b>CURRENTUSER</b>
 
 <br/>
 
-ADMINLINK
+<div>Logged in as: <b>CURRENTUSER</b> ADMINLINK</div>
+<br/>
 
 <p>
 <ul>
 
-<li> <a href="CGINAME?query=day_report.1&test_run=conda_%">CONDA</a>
-<li> <a href="CGINAME?query=day_report.2&test_run=jwst_latest">Latest JWST Report</a>
-<li> <a href="CGINAME?query=day_report.2&test_run=etc_hst_daily_latest">Latest HST pyetc</a>
-<li> <a href="CGINAME?query=day_report.2&test_run=etc_jwst_daily_latest">Latest Pandeia Report (Local)</a>
-<li> <a href="CGINAME?query=day_report.2&test_run=pandeia_master_latest">Latest Pandeia Report (AWS)</a>
-<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest">Latest Daily Report</a>
-	<ul>
-	<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest&project=astrolib">astrolib</a>
-	<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest&project=drizzlepac">drizzlepac</a>
-	<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest&project=hstcal">hstcal</a>
-	<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest&project=jwst">jwst</a>
-	<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest&project=stsci_python">stsci_python</a>
-	<li> <a href="CGINAME?query=day_report.2&test_run=daily_latest&project=stsdas">stsdas</a>
-	<li>
-	<form action=CGINAME method=GET>
-		<input type=hidden name=query value='day_report.2'>
-		<input type=text name=test_run value='daily_latest'>
-	</form>
-	<li> <a href="CGINAME?query=treewalk&test_run=daily_latest">Latest Daily Report </a> (tree walker)
-	</ul>
+<!--pretty sure we don't need this link, but I'm leaving it here for now -->
+<!--<li> <a href="CGINAME?query=day_report.2&test_run=jwst_latest">Latest JWST Report</a>-->
 
+<li> <a href="CGINAME?query=day_report.2&test_run=etc_hst_daily_latest"><b>Latest HST pyetc</b></a>
```

Comment:
This is great, I've been vaguely wanting to clean out all the old SSB cruft for a while.
I mentioned this in person today but it would be great to add the engine long link, as well as the py3 link (although that one might change shortly?).

Reply:
Filed https://github.com/spacetelescope/pandokia/issues/47.